modulus/distributed/manager.py

```diff
@@ -427,20 +430,20 @@ def create_process_subgroup(
     @staticmethod
     def create_orthogonal_process_group(
         name: str, group_name: str, verbose: bool = False
-    ):
+    ):  # pragma: no cover
         """
         Create a process group that is orthogonal to the specified process group.
 
         Parameters
         ----------
         name : str
```

Comment:
This is super minor, but from a code-readability perspective I wonder if `orthogonal_group_name` would be more descriptive (albeit, longer).

Reply:
Made this change